### ISSUE TITLE
space after open parenthesis

### ISSUE THREAD

**Issue by arademaker** [own-pt/sensetion.el#94]
> (n) production | *( law)* the act of exhibiting in a court of law ; the appellate court demanded the production of all documents ;

the space after the open parenthesis is not normal.

**Comment by odanoburu** [comment]
I guess there's no fool-proof way of doing the segmentation, so it's probably better to just separate all tokens by a space. is that okay?

**Comment by arademaker** [comment]
Yep. More consistent at least

**Comment by odanoburu** [comment]
fixed in the `es` branch.

**Comment by arademaker** [comment]
We started the work on `es` branch thinking about alternative backend,  now it does contain fixed not in the master branch... that is bad.

**Comment by odanoburu** [comment]
yes, but we can't really afford to have two branches developing in parallel, can we? I would like to merge the `es` branch on `master` as soon as we can..
